resources/keywords/deploy_instance_dev.resource

```diff
@@ -81,30 +81,34 @@ User Filters And Selects The Created Database Instance
     Click Button    ${btnConnectInst}
 
 Set ${type} View On Topology View
-    Wait Until Page Contains Element    ${btnDevTopologyViewSwitch}     timeout=10s
-    ${currentView}=     Get Element Attribute    ${btnDevTopologyViewSwitch}    aria-label
+    Wait Until Page Contains Element    ${btnDevTopViewSwitch}     timeout=10s
```

Comment:
${btnDevToplgyViewSwitch} --> ${btnDevToplgyViewSwitch} I don't think 'Top' will be understood as 'Topology'